src/utils.ts

```diff
@@ -289,6 +286,8 @@ export async function askToConfigureEnvironment(
 export async function askAndOpenProject(
     rootPath: string, workspaceFile: string,
     telemetryContext: TelemetryContext): Promise<void> {
+  telemetryContext.properties.result = TelemetryResult.Failed;
+
```

Comment:
Same here. Do we need to set it to 'failed' before user open the IoT Workspace project here?

Reply:
Same as above.